pmd-apex/src/main/resources/category/apex/bestpractices.xml

```diff
@@ -36,6 +36,31 @@ public class Foo {
         </example>
     </rule>
 
+    <rule name="DebugsShouldSpecifyLoggingLevel"
```

Comment:
We usually try to insert the rules sorted alphabetically by the rule name in the file (it's just a convention and helps, if you have only a text editor at hand). This means - if the rule name stays like this - it should be added later in the file (not between **A**pexAssertionsShouldIncludeMessage and **A**pexUnitTestClassShouldHaveAsserts, but after **A**voidLogicInTrigger).